js/models/userMd.js

```diff
@@ -92,6 +93,84 @@ module.exports = Backbone.Model.extend({
       return modGuid;
     });
 
+    // If blocked_guids has no legitmate guids, it returns an array with an empty string.
+    // Let's remedy that.
+    if (response.blocked_guids.length && response.blocked_guids[0] === "") {
+      response.blocked_guids = [];
+    }
+
     return response;
-  }
+  },
+
+  toJSON: function() {
```

Comment:
This is put in place to deal with the fact that the settings endpoint expects a 'blocked' key on POST, but on GETs it provides a 'blocked_guids' key. This adjustment should really happen in sync, but since we're not saving the model, rather toJSON'ing it to saveToAPI, I'm doing it here.